my_functions.py

```diff
@@ -39,4 +39,9 @@ def is_prime(num):
     for i in range(2, int(num**0.5) + 1):
         if num % i == 0:
             return False
-    return True
\ No newline at end of file
+    return True
+
+def reverse_string(s):
```

Comment:
Add a doc string explaining the function, its arguments and its return values.

Reply:
Added doc string

Reply:
Nice